### PR TITLE
Dan.add erc1155 tbc to cw app

### PR DESCRIPTION
**Pull request by 0xdantrinh**
## Description
Adding ERC1155 support for CW upon calling TBC within the tokenBalance internal route.

## How has this been tested?
Tested api/tokenBalance route to make sure it still works. Could still test the internal /tokenBalance route

### CODE REVIEW

**Comment by CowMuon** [comment]
Converted to draft. This needs to be reviewed in full. @zakhap

**Comment by dillchen** [comment]
Semi confused on the user story here:

- As an admin, I should be able to set (on the client side, gating any community / topic with an 1155)

To confirm:
- As an admin, I'm able to set multiple tokens within a community (as a result of Contracts 1 & 2 being merged in?) this "should be true"

**Comment by dantrinh** [comment]
As far as the user story was, adding this to create_community is not a part of this. This is just to add support for 1155 balances being able to be retrieved by calling the internal route within CWApp.

what you speak of can be added though! I am speaking with Kurtis tomorrow aka this week to go over this new backend/cache change so that this 1155 flow can be used in create_community

File: packages/common-common/src/types.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import { ERC1155 } from './eth/types/ERC1155.d';
```
[the remark this answers is not in the document]
Comment (dantrinh) [reply packages/common-common/src/types.ts]: I think this got auto imported by accident during my types enum adding!

**Pull request closed without merging**